.github/workflows/codespeed-benchmarks.yml

```diff
@@ -17,6 +17,7 @@ jobs:
   pre_job:
     name: Skip Duplicate Jobs Pre Job
     runs-on: ubuntu-latest
+    if: ${{ ! vars.UNIT_TESTS_DISABLED }}
```

Comment:
Sadly benchmarks (at least the publishing of results to CodeSpeed instance) has been broken for a while. Maybe we can just disable the steps which publish results to CodeSpeed, but leave everything else in place (so we still exercise micro benchmarks and other benchmarking related code).

Reply:
The benchmark work for me. Disabling Unit tests is meant for the internal repo only.

Reply:
Thanks for clarifying. In this case it would be good to document in that readme file (when disabling unit tests makes sense or is desired).

Reply:
Modified the description in README.md. Btw, I didn't mention the internal repo on purpose.